boot-qemu.sh

```diff
@@ -285,7 +285,7 @@ function invoke_qemu() {
             # Note: no -serial mon:stdio
             "${QEMU[@]}" \
                 "${QEMU_ARCH_ARGS[@]}" \
-                -append "${APPEND_STRING}" \
+                -append "${APPEND_STRING%* }" \
```

Comment:
this is the kind of thing where in 6 months you look at and go "WTF is that?" The use of sigils is a perl-ism that I despise.

Reply:
Fair, I can pull it up and add a comment. Bash parameter expansion is better than shelling out to external utilities though so I would prefer not to eliminate it.